src/App.tsx

```diff
@@ -1,61 +1,127 @@
+import React, { useState } from 'react';
+import { TodoList } from './components/TodoList/TodoList';
+import { Todo } from './Types/Todo';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { getUserById } from './Helpers/getUser';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { User } from './Types/User';
+
+export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>(todosFromServer);
+  const [title, setTitle] = useState<string>('');
+  const [userId, setUserId] = useState<number | string>('');
+  const [isTitleValid, setIsTitleValid] = useState<boolean>(true);
+  const [isUserSelected, setIsUserSelected] = useState<boolean>(true);
+
+  const todosWithUsers: Todo[] = todos.map(todo => ({
+    ...todo,
+    user: getUserById(todo.userId, usersFromServer),
+  }));
+
+  const addTodo = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (!title || !userId) {
+      if (!title) {
+        setIsTitleValid(false);
+      }
+
+      if (!userId) {
+        setIsUserSelected(false);
+      }
+
+      return;
+    }
+
+    const maxId = Math.max(...todos.map(todo => todo.id));
```

Comment:
Create helper function for get new todo id and move it to `helpers` and return new id in this function

Reply:
fixed 👌